packages/zoltan2/src/algorithms/connectivity/Zoltan2_VtxLabel.hpp

```diff
@@ -0,0 +1,378 @@
+#include "Tpetra_Core.hpp"
+#include "Teuchos_RCP.hpp"
+#include "Tpetra_Import.hpp"
+#include "Tpetra_FEMultiVector.hpp"
+#include "Tpetra_MultiVector_decl.hpp"
+#include "Tpetra_MultiVector_def.hpp"
+
+#include <string>
+#include <sstream>
+#include <iostream>
+#include <queue>
+#include <vector>
+
+#ifndef ZOLTAN2_VTXLABEL_
+#define ZOLTAN2_VTXLABEL_
+
+namespace Zoltan2{
+        
+        template<typename MAP>
+        class iceSheetPropagation;
```

Comment:
Why is this forward declaration needed?  I don't see any references to iceSheetPropagation in this file.

Reply:
I have removed this forward declaration.